src/ui/suspense/scopeCache.ts

```diff
@@ -1,37 +1,24 @@
-import { FrameId, MappedLocation, PauseId, Scope, SourceId } from "@replayio/protocol";
+import { FrameId, PauseId, Scope, SourceId } from "@replayio/protocol";
 
-import { createGenericCache } from "bvaughn-architecture-demo/src/suspense/createGenericCache";
-import { Pause, ThreadFront } from "protocol/thread";
-import { assert } from "protocol/utils";
+import {
+  FrameScopes,
+  getScopesAsync as getScopesAsyncNew,
+  getScopesIfCached as getScopesIfCachedNew,
```

Comment:
Eee 😬 How long will we have "getScopesIfCached" and "getScopesIfCachedNew"? That feels pretty confusing.

Reply:
Not for long! This is what I meant when I wrote
> I've left `get(Frame|Scope)(Suspense|Async|IfCached)` methods in the old place that call the corresponding new methods - removing them and calling the new methods directly is straightforward but touches a lot of files so I'll do that in a follow-up PR